redfish-core/src/redfish.cpp

```diff
@@ -164,6 +164,9 @@ RedfishService::RedfishService(App& app)
     requestRoutesPprStatus(app);
     requestRoutesPprFile(app);
 
+    requestRoutesOobErrorInjectionMode(app);
+    requestRoutesSupportedErrorTypes(app);
```

Comment:
use single Route function by changing route path /redfish/v1/Systems/<str>/Processors/<str>/Actions/Oem/{str} ? and calll approppriate d-bus api will similify the code.